src/components/TodoContentHeader.tsx

```diff
@@ -0,0 +1,111 @@
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { todosApi } from '../api/todos-api';
+import { useTodoContext } from '../context/todoContext/useTodoContext';
+import { useErrorContext } from '../context/errorContext/useErrorContext';
+import { Todo } from '../types/Todo';
+import { TextField } from './TextField';
+
+interface TodoContentHeaderProps {
+  setTempTodo: (todo: Todo | null) => void
+}
+
+export const TodoContentHeader: FC<TodoContentHeaderProps> = (props) => {
+  const { setTempTodo } = props;
+  const [title, setTitle] = useState('');
+  const [isHandleRequest, setIsHandleRequest] = useState(false);
+  const {
+    addTodo,
+    size,
+    countCompleted,
+    todos,
+    setHandlingTodoIds,
+    updateTodos,
+  } = useTodoContext();
+  const { notifyAboutError } = useErrorContext();
+
+  const submit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (!title) {
+      notifyAboutError("Title can't be empty");
+
+      return;
+    }
+
+    try {
+      setTempTodo({
+        id: 0,
+        title,
+        completed: false,
+        userId: 0,
+      });
+      setIsHandleRequest(true);
+      const createdTodo = await todosApi.create({
+        title,
+        completed: false,
+        userId: 10875,
+      });
+
+      addTodo(createdTodo);
+    } catch {
```

Comment:
In the `catch` block, you should also catch a possible `error`, because now the error does not look informative, for example, you can do this:
```
catch(error) { notifyAboutError(`Unable to add a todo: ${error.message}`)}
```
This applies to all code.